### DISCUSSION TITLE
iOS 替换文字时，文字能否设置居中对齐

### DISCUSSION THREAD

**Discussion by camp1990** [Tencent/libpag#1869] [Q&A] [answered]
![1782038-84c1b55393062b72](https://github.com/Tencent/libpag/assets/33602526/bc8019cb-ddef-4821-8247-39b3327bd6f2)
![image](https://github.com/Tencent/libpag/assets/33602526/e2d4a0a0-1c55-4f93-bc76-b6a0d1c27050)

**Comment by kevingpqi123** [comment, marked as answer]
默认就是居中对齐的，应该是文本框区域设置的问题，比如上面的“元抵现券”，文本框高度设置成和 “10”一样高
